config/config.json

```diff
@@ -1,9 +1,9 @@
 {
   "development": {
-    "username": "",
-    "password": "",
-    "database": "",
-    "host": "",
+    "username": "postgres",
+    "password": "postgres",
+    "database": "bsc",
+    "host": "localhost",
```

Comment:
Remove these details. It's sensitive information which should not be in version control